scripts/output_env.sh

```diff
@@ -77,15 +77,13 @@ echo
 print_version "cmake" "--version" "" "head -n 1"
 echo
 
-if [ "${RUN_ARMCC:-1}" -ne 0 ]; then
```

Comment:
Please keep the check on `RUN_ARMCC`. The goal of this check is to avoid timing out and erroring out if you run `all.sh` without access to a license.

Reply:
My typical scenario is: I'm working from home or on the move, and not on the VPN. I have `ARMLMD_LICENSE_FILE` set all the time because I do want Arm compilers to work when they can. Invoking `armcc --version` or the like makes it hang.

Also, even if it doesn't hang, it's slow. In particular, `all.sh --list-components` should not invoke an Arm compiler, otherwise it makes completion painfully slow.